jaeger-spark-dependencies-elasticsearch/src/main/java/io/jaegertracing/spark/dependencies/elastic/ElasticsearchDependenciesJob.java

```diff
@@ -60,6 +60,8 @@ public static final class Builder {
     Boolean nodesWanOnly = Boolean.parseBoolean(Utils.getEnv("ES_NODES_WAN_ONLY", "false"));
     String indexPrefix = Utils.getEnv("ES_INDEX_PREFIX", null);
     String spanRange = Utils.getEnv("ES_TIME_RANGE", "24h");
+    Boolean useReadAlias = Boolean.parseBoolean(Utils.getEnv("ES_USE_READ_ALIAS", "false"));
+    Boolean useWriteAlias = Boolean.parseBoolean(Utils.getEnv("ES_USE_WRITE_ALIAS", "false"));
```

Comment:
Do we need two env variables for read and write? I would say that if read alias is used then write should be used as well. 

Also please document this in the root readme

Reply:
Done. PTAL.